### PR TITLE
Fix for discussion title / description events not being published

### PR DESCRIPTION
**Pull request by gdpelican**
Well I didn't mean to work on a beta feature, but this one lept out of the abyss and gnawed on me during some angular stuff. :whale:

### CODE REVIEW

File: spec/services/discussion_service_spec.rb [hunk 1]
```diff
@@ -1,26 +1,8 @@
 require 'rails_helper'
 
 describe 'DiscussionService' do
-  let(:ability) { double(:ability, :authorize! => true, can?: true) }
-  let(:user) { double(:user, ability: ability, update_attributes: true) }
```
Comment (gdpelican) [inline spec/services/discussion_service_spec.rb]: Double testing is like constructing your own little fantasy world in which you are the god of all unicorns and your code works. :goat:

**Comment by gdpelican** [comment]
Hey @robguthrie would you mind giving this a once-over and merging? Would be good to get this fix in.

**Comment by robguthrie** [comment]
Looks good to me. Does this mean that fire_edit_title_event and fire_edit_description_event can be removed from discussion.rb (the might have been unused for a while now)

**Comment by gdpelican** [comment]
Yeah those are orphans. I took em out.

**Comment by gdpelican** [comment]
I want to merge this, @robguthrie. Any qualms?

**Comment by robguthrie** [comment]
looks good! :+1:

**Comment by rdbartlett** [comment]
Qualmless in Seattle

On 16 February 2015 at 15:11, James Kiesel <notifications@github.com> wrote:

> Merged #1879 <https://github.com/loomio/loomio/pull/1879>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/loomio/loomio/pull/1879#event-236083848>.
>



-- 

Richard D. Bartlett
Loomio co-founder
rich@loomio.org

<http://loomio.org/>

**Pull request merged**